API/chat/tests.py

```diff
@@ -670,13 +617,9 @@ def test_delete_message_that_does_not_exist(self):
             timestamp=timestamp
         )
 
-        qstring = urllib.urlencode({
-            'id': message.id + 1
-        })
-
         response = self.client.delete(
-            reverse('chat:message', args=(self.channel.name,)),
-            qstring
+            reverse('chat:message', args=(message.id + 1,)),
+            {}
```

Comment:
If we change the URL behaviors, we must change the ARCH document and the client also. Otherwise it's an implementation detail.